lib/collision/AABB.js

```diff
@@ -125,6 +127,7 @@ AABB.prototype.contains = function(aabb) {
 
 AABB.prototype.extend = function(value) {
   AABB.extend(this, value);
+  return this;
```

Comment:
Can you please update the return type of this method from `void` to `AABB` in the TS definitions?